src/main/java/Duke.java

```diff
@@ -1,10 +1,174 @@
+import java.io.*;
```

Comment:
please remember that when importing files, list them instead of using the wildcard *. It helps when checking for errors. Also, avoid code where most of the method is in the client class and the main method.